src/main.rs

```diff
@@ -187,7 +187,7 @@ impl MPDLibrary {
 
     fn mpd_to_bliss_path(&self, mpd_song: &MPDSong) -> Result<PathBuf> {
         let file = &mpd_song.file;
-        let path = if file.to_lowercase().contains(".cue/track") {
+        let path = if file.to_lowercase().contains(".cue/track") || file.to_lowercase().contains(".flac/track") {
```

Comment:
have you tried this / did it work out of the box? I'm curious if bliss is going to handle that properly :)

Reply:
oh ... sorry for leaving this in the pull request, it does not belong here (more to #54).

I tried it, but bliss fails on it with ffmpeg complaining like this:

```shell
[2024-05-12T18:21:48Z ERROR bliss_audio::library] Analysis of song '/some/dir/example.flac/track0008' failed: error happened while decoding file – while opening format for file '/some/dir/example.flac/track0008': ffmpeg::Error(20: Not a directory). The error has been stored.
```

Reply:
ahh, makes sense :) would it be possible to send an example of such files, so I can try and see if I can implement it myself? And remove this change in the meantime so I can merge the wrapper thing 😁